### DISCUSSION TITLE
Neutral Color Palette

### DISCUSSION THREAD

**Discussion by Sagie501** [ant-design/ant-design#45411] [Q&A]
In Antd colors doc, there is a [section](https://ant.design/docs/spec/colors#neutral-color-palette) regarding neutral color palette.

In this section there are 13 colors for the neutral colors (shades of gray colors).

How ever I didn't understand how to use these colors. All the other base color palettes like dust red, volcano etc, are available via the `theme.useToken()`. But the neutral colors aren't there.

How can I use these colors? What I am missing?

**Comment by Sagie501** [comment]
Any one help?

**Reply by matthiasferch** [reply]
I have the same issue. Our designer makes heavy use of the neutral color palette and I don't know how to access them from within antd.

**Comment by restareaByWeezy** [comment]
having same issue any help?

**Comment by andyxmai** [comment]
Can we add those neutral color tokens to the libary? E.g `token.gray-1`, `token.gray-2`, etc.

**Comment by remkodejong** [comment]
I'm also wondering where this palette needs to come from? We have a `gray` and a `grey` color in `@ant-design/colors` package which are identical, but not the same as the "Neutral Palette".

**Comment by niti2539** [comment]
I have the same issue.
Solved by:  manually add gray color palette to the theme configuration

```
  <ConfigProvider
    theme={{
      cssVar: true // allow mapping color to css variable and use in App
      token: {
        // Seed Token
        colorPrimary: '#00b96b',
        borderRadius: 2,

        // gray color palette
        gray1: '#ffffff',
        .
        .
        .
        gray13: '#000000'
      },
    }}
  >
```
and use it like this:

`
   background: var(--ant-gray-1);
`

or
` background: token.gray1`

gray palette from antd https://ant.design/docs/spec/colors#neutral-color-palette

Solution from : https://stackoverflow.com/a/77127362/9442224
